ccx_messaging/downloaders/http_downloader.py

```diff
@@ -70,7 +70,8 @@ class HTTPDownloader:
     HTTP_RE = re.compile(
         r"^(?:https://[^/]+\.s3\.amazonaws\.com/[0-9a-zA-Z/\-]+|"
         r"https://s3\.[0-9a-zA-Z\-]+\.amazonaws\.com/[0-9a-zA-Z\-]+/[0-9a-zA-Z/\-]+|"
-        r"http://minio:9000/insights-upload-perma/[0-9a-zA-Z\.\-]+/[0-9a-zA-Z\-]+)\?"
```

Comment:
This regex was probaly here  to allow devs to run this with a local minio instance